Extract/Extract.ipynb

```diff
@@ -0,0 +1,264 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "root = Tk()\n",
+    "root.withdraw()\n",
+    "file_path = askopenfilename()\n",
+    "f = open(file_path)\n",
+    "text = f.read()\n",
+    "root.update()\n",
+    "root.destroy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Select the exsisting annotations you want to upload\n",
+    "This will open a file picker in a new window"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "root = Tk()\n",
+    "root.withdraw()\n",
+    "csv_file_path = askopenfilename()\n",
+    "df = pd.read_csv(csv_file_path)\n",
+    "root.update()\n",
+    "root.destroy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Find downloads folder windows, linux, mac"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if os.name == 'nt':\n",
+    "    import ctypes\n",
+    "    from ctypes import windll, wintypes\n",
+    "    from uuid import UUID\n",
+    "\n",
+    "    # ctypes GUID copied from MSDN sample code\n",
+    "    class GUID(ctypes.Structure):\n",
+    "        _fields_ = [\n",
+    "            (\"Data1\", wintypes.DWORD),\n",
+    "            (\"Data2\", wintypes.WORD),\n",
+    "            (\"Data3\", wintypes.WORD),\n",
+    "            (\"Data4\", wintypes.BYTE * 8)\n",
+    "        ] \n",
+    "\n",
+    "        def __init__(self, uuidstr):\n",
+    "            uuid = UUID(uuidstr)\n",
+    "            ctypes.Structure.__init__(self)\n",
+    "            self.Data1, self.Data2, self.Data3, \\\n",
+    "                self.Data4[0], self.Data4[1], rest = uuid.fields\n",
+    "            for i in range(2, 8):\n",
+    "                self.Data4[i] = rest>>(8-i-1)*8 & 0xff\n",
+    "\n",
+    "    SHGetKnownFolderPath = windll.shell32.SHGetKnownFolderPath\n",
+    "    SHGetKnownFolderPath.argtypes = [\n",
+    "        ctypes.POINTER(GUID), wintypes.DWORD,\n",
+    "        wintypes.HANDLE, ctypes.POINTER(ctypes.c_wchar_p)\n",
+    "    ]\n",
+    "\n",
+    "    def _get_known_folder_path(uuidstr):\n",
+    "        pathptr = ctypes.c_wchar_p()\n",
+    "        guid = GUID(uuidstr)\n",
+    "        if SHGetKnownFolderPath(ctypes.byref(guid), 0, 0, ctypes.byref(pathptr)):\n",
+    "            raise ctypes.WinError()\n",
+    "        return pathptr.value\n",
+    "\n",
+    "    FOLDERID_Download = '{374DE290-123F-4565-9164-39C4925E467B}'\n",
+    "\n",
+    "    def get_download_folder():\n",
+    "        return _get_known_folder_path(FOLDERID_Download)\n",
+    "else:\n",
+    "    def get_download_folder():\n",
+    "        home = os.path.expanduser(\"~\")\n",
+    "        return os.path.join(home, \"Downloads\")"
```

Comment:
Instead of all that, just ask the user for a folder where they want to store all the resulting csv files. This should reduce the code here and possibility for errors.

Reply:
removed and changed to a variable where users can paste in a folder path